src/MyCocktails/MyCocktails.tsx

```diff
@@ -9,17 +9,14 @@ const MyCocktails: React.SFC<MyCocktailsProps> = () => {
 	const [allCocktails, setAllCocktails] = useState({})
 
 	// NonFunctional code to connect array of cocktails
-	// const displayingAllCocktails = async () => {
```

Comment:
Really nice moving the API's back to App, makes sooo much sense after hearing your mentor explain why!